services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/application.py

```diff
@@ -43,7 +42,7 @@ def assemble_application() -> FastAPI:
     needed in other requests and used to share data.
     """
 
-    dynamic_sidecar_settings = DynamicSidecarSettings.create()
+    dynamic_sidecar_settings = DynamicSidecarSettings.create_from_envs()
```

Comment:
I thought this was the actual standard.

Reply:
the standard is using ```DynamicSidecarSettings()``` as far as I know

Reply:
All **settings** inherited from the new ``settings_library.BaseCustomSettings`` must be initialized using ``create_from_envs`` instead of the constructor to overcome a limitation with "sub settings" defaults. For more details see: https://github.com/ITISFoundation/osparc-simcore/blob/985d8a469b1c9860adce65237b7a61bbaaae5ac0/packages/settings-library/src/settings_library/base.py#L58